tools/wave/tox.ini

```diff
@@ -1,14 +1,13 @@
 [tox]
-envlist = py38,py39,py310,py311,py312
+envlist = py36,py37,py38,py39
```

Comment:
[prior commits to this file](https://github.com/web-platform-tests/wpt/commits/master/tools/wave/tox.ini) from the past couple years indicate this `envlist` change should be reverted